src/upgrades.ts

```diff
@@ -1,71 +1,16 @@
-import { faderValueToDB } from 'soundcraft-ui-connection';
-import InstanceSkel = require('../../../instance_skel');
-import {
-  CompanionCoreInstanceconfig,
-  CompanionMigrationAction,
-  CompanionMigrationFeedback,
-  CompanionUpgradeContext
-} from '../../../instance_skel_types';
-import { ActionType } from './actions';
-import { UiConfig } from './config';
-import { FeedbackType } from './feedback';
-
-// see https://github.com/bitfocus/companion/wiki/Migrating-legacy-to-boolean-feedbacks
-export const upgradeLegacyFeedbackToBoolean = InstanceSkel.CreateConvertToBooleanFeedbackUpgradeScript({
-  [FeedbackType.MuteMasterChannel]: true,
-  [FeedbackType.SoloMasterChannel]: true,
-  [FeedbackType.DimMaster]: true,
-  [FeedbackType.MuteAuxChannel]: true,
-  [FeedbackType.PostAuxChannel]: true,
-  [FeedbackType.MuteFxChannel]: true,
-  [FeedbackType.PostFxChannel]: true,
-  [FeedbackType.MediaPlayerState]: true,
-  [FeedbackType.MediaPlayerShuffle]: true,
-  [FeedbackType.DTRecordState]: true,
-  [FeedbackType.MuteMuteGroup]: true
-});
-
-export function upgradeV2x0x0(
```

Comment:
This update script was to migrate from the ancient v1 of this module that was written in JS.
My intention for removal was that everyone who was still using the module should have been upgraded to v2 even before going to v3.

Reply:
I have no problem with that, but you will need to replace it with an empty function.
We track whether they have run by recording the count that have been run. So with it going from 2 to 1 upgrade scripts, some users will have their db reporting that they have run 2, some will report that they have run 1. When you add a new upgrade script, all those whose db says 2 will think they have already run it.

Reply:
provided a dummy script: fa49dfffd27845a5e5fc3dba